src/backend/joanie/tests/core/test_api_admin_course_runs.py

```diff
@@ -276,7 +476,7 @@ def test_admin_api_course_run_read_list_authenticated_with_nested_course_filter_
         self.assertEqual(content["count"], 1)
         self.assertEqual(content["results"][0]["id"], str(course_run.id))
         response = self.client.get(
-            f"/api/v1.0/admin/courses/{course.id}/course-runs/?query=Cour",
+            f"/api/v1.0/admin/courses/{course.id}/course-runs/?query=CourseRu",
```

Comment:
I don't understand why this is modified ?

Reply:
That's because the query filter has been updated to looking for resource_link.

Then the CourseRunFactory generates sequenced resource_link and the url contains the word courses.

So as the query filter is not case sensitive, with the update, in this test, all course runs were return so I updated the test to only match course run title as before.